### PR TITLE
chore: Docs add model version and api version

### PR DESCRIPTION
**Pull request by ZhongpinWang**
## Context

AI/gen-ai-hub-sdk-js-backlog#139.

Add documentation for

 - setting `model_version` in orchestration
 - the used Azure OpenAI API version
 - setting `requestConfig` in
   - Azure OpenAI client
   - orchestration client

## Definition of Done

- ~[ ] Code is tested (Unit, E2E)~
- ~[ ] Error handling created / updated & covered by the tests above~
- [X] Documentation updated
- ~[ ] (Optional) Aligned changes with the Java SDK~
- ~[ ] (Optional) Release notes updated~

### CODE REVIEW

File: packages/foundation-models/README.md [hunk 1]
```diff
@@ -60,26 +60,42 @@ const chatClient = new AzureOpenAiChatClient({
...
 
-### Azure OpenAI Client
+### Chat Client
 
-The Azure OpenAI client can then be used to send chat completion or embedding requests to models deployed in the SAP generative AI hub.
+Use the `AzureOpenAiChatClient` to send chat completion requests to an OpenAI model deployed in SAP generative AI hub.
 
-#### Chat Client
+The client sends request with Azure OpenAI API version `2024-06-01`.
+Set a different version together with other request configuration in the `requestConfig` parameter.
```
Comment (deekshas8) [inline packages/foundation-models/README.md:68]: [q] If user sets a newer version, are we sure that it works with our client? 
Can we instead say something like this client is generates using api version spec so and so, and is backward compatible (I imagine it is??).

File: packages/foundation-models/README.md [hunk 2]
```diff
@@ -60,26 +60,42 @@ const chatClient = new AzureOpenAiChatClient({
...
+    messages: [
+      {
+        role: 'user',
+        content: 'Where is the deepest place on earth located'
+      }
+    ]
+  },
+  {
+    headers: {
+      'content-type': 'application/json',
```
Comment (deekshas8) [inline packages/foundation-models/README.md:87]: @KavithaSiva already added a similar section in her PR. Please ensure we dont end up with dups

File: packages/foundation-models/README.md (hunk 1 above)
Comment (ZhongpinWang) [reply packages/foundation-models/README.md:68]: I will remove line 68. It also feels strange to have a different API version in the header, as our client is generated.

File: packages/foundation-models/README.md (hunk 2 above)
Comment (ZhongpinWang) [reply packages/foundation-models/README.md:86]: This falls under the scope of the issue. I had a look into the clean up PR and see a section documenting custom config only in orchestration. @KavithaSiva I think the easiest way is to remove the section you created in #159

File: packages/foundation-models/README.md [hunk 3]
```diff
@@ -60,26 +60,40 @@ const chatClient = new AzureOpenAiChatClient({
...
```
Comment (deekshas8) [inline packages/foundation-models/README.md:86]: [req] Aren't these headers already sent by default anyway? Can we just give an example of a custom dummy header? `content-type`  and `ai-client-type` are passed with these values, and `ai-resource-group` is meant to be set at the time of initialization. Setting it here is technically incorrect.

Comment (ZhongpinWang) [reply packages/foundation-models/README.md]: True. Thanks for pointing it out.

**Review by deekshas8 — APPROVED** [review]
LGTM.

File: packages/orchestration/README.md [hunk 4]
```diff
@@ -61,9 +81,20 @@ const orchestrationClient = new OrchestrationClient({
...
+  {
+    headers: {
+      // Add more headers here
+    },
+    params: {
+      // Add more parameters here
+    }
+    // Add more request configuration here
+  }
+);
```
[the remark this answers is not in the document]
Comment (ZhongpinWang) [reply packages/orchestration/README.md:97]: #191

**Pull request merged**